Apromore-Boot/src/main/resources/menus.json

```diff
@@ -76,6 +76,9 @@
             {
               "id": "org.apromore.plugin.portal.loganimation2.LogAnimationPlugin"
             },
+            {
+              "id": "org.apromore.plugin.portal.loganimation_ee.LogAnimationPlugin"
```

Comment:
Is it possible to have a package structure similar to `org.apromore.ee.plugin.portal.loganimation.LogAnimationPlugin` ?

Reply:
Done